extensions/positron-python/python_files/posit/positron/data_explorer.py

```diff
@@ -1519,6 +1527,11 @@ def _get_single_column_schema(self, column_index: int):
     def _get_column_name(self, index: int):
         return str(self.table.columns[index])
 
+    def _get_column_type_display(self, column_index: int) -> ColumnDisplayType:
+        column = self.table.iloc[:, column_index]
+        type_name, _ = self._get_type(column, column_index, self.state)
```

Comment:
```suggestion
       _, type_display = self._get_type(column, column_index, self.state)
       return type_display
```

`_get_type` returns:

https://github.com/posit-dev/positron/blob/be9250ef79dfeb41dbbf90a0a026f3dc07288823/extensions/positron-python/python_files/posit/positron/data_explorer.py#L1441

where the `type_display` is already the output of `_get_type_display(type_name)`

Reply:
fixing, thank you